src/js/actions/history.js

```diff
@@ -104,10 +105,13 @@ define(function (require, exports) {
                 });
             })
             .then(function () {
-                var currentDocument = this.flux.store("application").getCurrentDocument();
+                var document = this.flux.store("application").getCurrentDocument(),
```

Comment:
I confirmed that the catch clause is necessary because when no layers are selected, the `getProperty` throws an error.  I updated the comment.  I will admit to having stared at that comment before....and then blissfully ignoring it.  So, thanks.